### PR TITLE
styling invite code page. 

### PR DESCRIPTION
**Pull request by chrisnojima**
fixed container and etc. be careful when using the absolute position, since we don't have z-index etc we need to make sure its in flow order else you'll lose the press events

@keybase/react-hackers

### CODE REVIEW

File: shared/common-adapters/back-button.native.js [hunk 1]
```diff
@@ -19,10 +20,12 @@ export default class BackButton extends Component {
 
   render () {
     return (
-      <Box style={{...styles.container, ...this.props.style}} onClick={e => this.onClick(e)}>
```
Comment (chrisnojima) [inline shared/common-adapters/back-button.native.js]: onClick -> onPress and you can't add a handler like that on a raw View/Box, have to use a Touchable* view

File: shared/local-debug.native.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ let config = {
   overrideLoggedInTab: null
 }
 
-if (__DEV__ && false) {
+if (__DEV__ && true) {
```
Comment (chrisnojima) [inline shared/local-debug.native.js]: while we're devving this out maybe we should leave this on, then we get nicer actions logging

File: shared/login/forms/container.native.js [hunk 3]
```diff
@@ -7,10 +7,10 @@ import type {Props} from './container'
 export default ({children, onBack, style, outerStyle}: Props) => {
   return (
     <Box style={{...styles.container, ...outerStyle}}>
-      {onBack && <BackButton style={styles.button} onClick={onBack}/>}
       <Box style={{...styles.innerContainer, ...style}}>
         {children}
       </Box>
+      {onBack && <BackButton style={styles.button} onClick={onBack}/>}
```
Comment (chrisnojima) [inline shared/login/forms/container.native.js]: if its first it'll be 'below' the Box so it won't get presses

File: shared/login/index.js [hunk 4]
```diff
@@ -37,7 +37,8 @@ function loginRouter (currentPath: Map<string, string>, uri: URI): any {
   return {
     componentAtTop: {
       element,
-      hideBack: true
+      hideBack: true,
```
Comment (chrisnojima) [inline shared/login/index.js]: need this to hide the nav back button

File: shared/common-adapters/back-button.native.js [hunk 5]
```diff
@@ -19,10 +20,12 @@ export default class BackButton extends Component {
...
-        <Icon type='fa-arrow-left' style={styles.icon}/>
-        <Text type='BodyPrimaryLink' onClick={e => this.onClick(e)}>Back</Text>
-      </Box>
+      <TouchableWithoutFeedback onPress={e => this.onClick(e)}>
+        <Box style={{...styles.container, ...this.props.style}} >
```
Comment (MarcoPolo) [inline shared/common-adapters/back-button.native.js]: I got an error when doing this with TouchableHighlight that Box should forward `setNativeProps`, do you not get that error here?

I basically changed Box to be
```
import {View} from 'react-native'
export default View
```

**Comment by MarcoPolo** [comment]
👍 % comment

File: shared/common-adapters/back-button.native.js (hunk 5 above)
Comment (chrisnojima) [inline shared/common-adapters/back-button.native.js]: i think i didn't run into this as it only happens if you wrap it with feedback (vs w/o)

Comment (chrisnojima) [inline shared/common-adapters/back-button.native.js]: we can change the view to do this, i'll do it when the branches off of this branch get merged down

**Pull request merged**